Lib/fontTools/varLib/interpolate_layout.py

```diff
@@ -60,26 +60,41 @@ def interpolate_layout(designspace, loc, master_finder=lambda s:s, mapped=False)
 def main(args=None):
 	"""Interpolate GDEF/GPOS/GSUB tables for a point on a designspace"""
 	from fontTools import configLogger
-
+	import argparse
 	import sys
+
 	if args is None:
 		args = sys.argv[1:]
```

Comment:
we don't need this if block anymore, because when you pass `None` to parse_args it automatically does exactly that